frontend/appflowy_flutter/lib/mobile/presentation/database/field/mobile_field_type_option_editor.dart

```diff
@@ -446,9 +450,10 @@ class _PropertyType extends StatelessWidget {
             return DraggableScrollableSheet(
               expand: false,
               snap: true,
-              initialChildSize: 0.7,
-              minChildSize: 0.7,
-              builder: (context, controller) => FieldOptions(
+              initialChildSize: 0.97,
+              minChildSize: 0.97,
+              maxChildSize: 0.97,
```

Comment:
Provide a screenshot so we can see the UI style with 0.97 size factor.

Reply:
<img width="484" alt="Screenshot 2024-01-29 at 11 33 06 AM" src="https://github.com/AppFlowy-IO/AppFlowy/assets/71320345/d3e8bd69-e6c6-46dc-bc7e-0edcfb41f525">

Reply:
The design has a black background and thus doesn't match the screenshot. I've raised this on Figma